libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6SE_H743/README.md

```diff
@@ -0,0 +1,214 @@
+|  4   | BAT_VOLTAGE_ADC | +3.3V |
+|  5   |       GND       |  GND  |
+|  6   |       GND       |  GND  |
+
+**TELEM1&TELEM2**
+
+| Pin  | Signal  | Volt  |
+| :--: | :-----: | :---: |
+|  1   |   VCC   |  +5V  |
+|  2   | UART_TX | +3.3V |
+|  3   | UART_RX | +3.3V |
+|  4   |   CTS   | +3.3V |
+|  5   |   RTS   | +3.3V |
+|  6   |   GND   |  GND  |
+
+**ADC**
+
+| Pin  | Signal  | Volt  |
+| :--: | :-----: | :---: |
+|  1   |   VCC   |  +5V  |
+|  2   | ADC_3V3 | +3.3V |
+|  3   | ADC_6V6 | +6.6V |
+|  4   |   GND   |  GND  |
+
+**SPI**
+
+| Pin  |  Signal  | Volt  |
+| :--: | :------: | :---: |
+|  1   |   VCC    |  +5V  |
+|  2   | SPI_SCK  | +3.3V |
+|  3   | SPI_MISO | +3.3V |
+|  4   | SPI_MOSI | +3.3V |
+|  5   |  SPI_CS  | +3.3V |
+|  6   |   GND    |  GND  |
+
+**I2C**
+
+| Pin  | Signal  | Volt  |
+| :--: | :-----: | :---: |
+|  1   |   VCC   |  +5V  |
+|  2   | I2C_SCL | +3.3V |
+|  3   | I2C_SDA | +3.3V |
+|  4   |   GND   |  GND  |
+
+**CAN1&CAN2**
+
+| Pin  | Signal | Volt  |
+| :--: | :----: | :---: |
+|  1   |  VCC   |  +5V  |
+|  2   | CAN_P  | +3.3V |
+|  3   | CAN_N  | +3.3V |
+|  4   |  GND   |  GND  |
+
+**GPS1**
+
+| Pin  | Signal  | Volt  |
+| :--: | :-----: | :---: |
+|  1   |   VCC   |  +5V  |
+|  2   | UART_TX | +3.3V |
+|  3   | UART_RX | +3.3V |
+|  4   | I2C_SCL | +3.3V |
+|  5   | I2C_SDA | +3.3V |
+|  6   |   GND   |  GND  |
+
+**GPS2&SAFETY**
+
+| Pin  |    Signal     | Volt  |
+| :--: | :-----------: | :---: |
+|  1   |      VCC      |  +5V  |
+|  2   |    UART_TX    | +3.3V |
+|  3   |    UART_RX    | +3.3V |
+|  4   |    I2C_SCL    | +3.3V |
+|  5   |    I2C_SDA    | +3.3V |
+|  6   |   SAFETY_SW   | +3.3V |
+|  7   | SAFETY_SW_LED | +3.3V |
+|  8   |    3V3_OUT    | +3.3V |
+|  9   |    BUZZER     | +3.3V |
+|  10  |      GND      |  GND  |
+
+**DEBUG**
+
+| Pin  | Signal | Volt  |
+| :--: | :----: | :---: |
+|  1   |  VCC   |  +5V  |
+|  2   |   TX   | +3.3V |
+|  3   |   RX   | +3.3V |
+|  4   | SWDIO  | +3.3V |
+|  5   | SWCLK  | +3.3V |
+|  6   |  GND   |  GND  |
+
+**SAFETY**
+
+| Pin  |    Signal     | Volt  |
+| :--: | :-----------: | :---: |
+|  1   |    3V3_OUT    | +3.3V |
+|  2   |   SAFETY_SW   | +3.3V |
+|  3   | SAFETY_SW_LED | +3.3V |
+|  4   |   SBUS_OUT    | +3.3V |
+|  5   |     RSSI      | +3.3V |
+|  6   |      GND      |  GND  |
+
+**USB EX**
+
+| Pin  | Signal | Volt  |
+| :--: | :----: | :---: |
+|  1   | VCC_IN |  +5V  |
+|  2   |   DM   | +3.3V |
+|  3   |   DP   | +3.3V |
+|  4   |  GND   |  GND  |
+
+## UART Mapping
+
+ - SERIAL0 -> USB (OTG1)
+ - SERIAL1 -> USART1 (Telem1)
+ - SERIAL2 -> USART6 (Telem2)
+ - SERIAL3 -> USART3 (GPS1), NODMA
+ - SERIAL4 -> USART2 (GPS2), NODMA
+ - SERIAL5 -> UART8 (SBUS)
+ - SERIAL6 -> UART7 (Debug), NODMA
+ - SERIAL7 -> USB2 (OTG2)
+
+## RC Input
+
+The remote control signal should be connected to the “RC IN” pin, at one side of the servo channels.
+
+This signal pin supports two types of remote control signal inputs, SBUS and PPM signals.

```

Comment:
this supports all unidirectional protocols...PPM,SBus,DSM,Fport(RC only),SRXL,etc